packages/os/os.spec

```diff
@@ -591,4 +596,8 @@ install -p -m 0644 %{S:301} %{buildroot}%{_cross_udevrulesdir}/81-ebs-volumes.ru
 %{_cross_unitdir}/bootstrap-containers@.service
 %{_cross_tmpfilesdir}/bootstrap-containers.conf
 
+%if %{with vmware_platform}
+%{_cross_unitdir}/disable-udp-offload.service
+%endif
```

Comment:
This places the file in the `bootstrap-containers` subpackage, which doesn't seem correct - it's unrelated to bootstrap containers.

I don't think there's an amazing spot for it in the existing subpackages. Maybe netdog?

Reply:
:facepalm:   I overlooked that - I'll get it up in the netdog subpackage.